apps/cookbook/src/app/component-overview/component-display/component-display.component.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { ComponentFixture, TestBed } from '@angular/core/testing';
```

Comment:
Consider adding more tests, instead of just the default generated tests :)

Reply:
As we figured out we don't run test in cookbook 😄